deploy/cloudformation/README.md

```diff
@@ -19,7 +19,6 @@ FLEET_URL="<Elastic Agent Fleet URL>"
 ENROLLMENT_TOKEN="<Elastic Agent Enrollment Token>"
 ELASTIC_ARTIFACT_SERVER="https://artifacts.elastic.co/downloads/beats/elastic-agent" # Replace artifact URL with a pre-release version (BC or snapshot)
 ELASTIC_AGENT_VERSION="<Elastic Agent Version>" # e.g: 8.8.0 | 8.8.0-SNAPSHOT
-# INTEGRATION=CloudSecurityPostureManagement # Defaults to VulnerabilityManagement if not specified
```

Comment:
Document the new parameter instead

Reply:
The DeploymentType is not a template parameter and is only being used by the `config.go`.